requirements.txt

```diff
@@ -86,3 +86,6 @@ pytest-flask ~=1.2
 jsonschema==4.17.3
 
 loguru
+
+# For scraping all possible metadata relating to products so we can do better restock detection
+extruct
```

Comment:
I still think just a str check to see if that data is available is better for basic checks before offering the change